src/components/sbb-loading-indicator/sbb-loading-indicator.scss

```diff
@@ -0,0 +1,235 @@
+@use '../../global/styles' as sbb;
+
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include sbb.host-component-properties;
+
+:host([variant='circle']) {
+  --sbb-loading-indicator-height: var(--sbb-size-icon-ui-small);
+  --sbb-loading-indicator-width: var(--sbb-size-icon-ui-small);
+  --sbb-loading-indicator-padding: var(--sbb-border-width-2x);
+  --sbb-loader-indicator-duration: 1s;
+
+  display: inline-block;
+  line-height: 0;
+}
+
+:host([variant='circle']) .sbb-loading-indicator {
+  height: var(--sbb-loading-indicator-height);
+  width: var(--sbb-loading-indicator-width);
+  padding-inline: var(--sbb-loading-indicator-padding);
+  padding-block: var(--sbb-loading-indicator-padding);
+  display: inline-block;
+}
+
+:host([variant='circle']) .sbb-loading-indicator__animated-element {
+  width: 100%;
+  height: 100%;
+  display: inline-block;
+  color: transparent;
+  position: relative;
+  margin: 0 auto;
+  overflow: hidden;
+  border-radius: 50%;
+  background: conic-gradient(from 90deg, #fff, #f00);
+  /* stylelint-disable */
+  -webkit-mask: radial-gradient(circle 7px, #0000 98%, #fff);
+  /* stylelint-enable */
+  mask: radial-gradient(circle 7px, #0000 98%, #fff);
+  animation: rotation var(--sbb-loader-indicator-duration) infinite linear;
+
+  &::after {
+    content: '';
+    width: 3px;
+    height: 3px;
+    background: #f00;
+    border-radius: 50%;
+    position: absolute;
+    top: 50%;
+    right: 0;
+    transform: translateY(-50%);
+    overflow: hidden;
+    margin: auto;
+  }
+}
+
+:host([variant='window']) {
+  --sbb-loading-indicator-height: sbb.px-to-rem-build(32);
+  --sbb-loading-indicator-width: sbb.px-to-rem-build(55);
+  --sbb-loading-indicator-padding: 0;
+  --sbb-loader-indicator-duration: 0.23s;
+
+  display: inline-block;
+  line-height: 0;
+}
+
+:host([variant='window'][size='small']) {
+  --sbb-loader-indicator-window-height: 1.66665em;
+  --sbb-loader-indicator-window-padding-block-start: 0.66666em;
+  --sbb-loader-indicator-window-element-width: 3.46665em;
+  --sbb-loader-indicator-window-element-height: 0.33333em;
+  --sbb-loader-indicator-window-element-perspective: 6em;
+  --sbb-loader-indicator-window-element-animation: loading-container-small
+    var(--sbb-loader-indicator-duration) linear infinite;
+  --sbb-loader-indicator-window-last-div-width: 0.53333em;
+  --sbb-loader-indicator-window-last-div-height: 0.33333em;
+  --sbb-loader-indicator-window-last-div-margin: 0.2em;
+  --sbb-loader-indicator-window-last-div-transform: translate3d(-0.1em, 0, 0);
+}
+
+:host([variant='window'][size='large']) {
+  --sbb-loader-indicator-window-height: 3em;
```

Comment:
Would it be possible to convert all the size units and use the `sbb.px-to-rem-build` function to be more consistent?

Reply:
Done in [fa07175](https://github.com/lyne-design-system/lyne-components/pull/1968/commits/fa0717537f50ac17552ce6ea1fe1136c757f7b27) after a call with Lukas